src/testshade/testshade.cpp

```diff
@@ -571,6 +613,7 @@ getargs (int argc, const char *argv[])
                 "--saveptx", &saveptx, "Save the generated PTX (OptiX mode only)",
                 "--warmup", &warmup, "Perform a warmup launch",
                 "--path %s", &shaderpath, "Specify oso search path",
+                "--library %s", &librarypath, "Specify library search path",
```

Comment:
Now that we have multiple kinds of search paths, I think `--path` is ambiguous and confusing. Grep says that we never even use the `--path` option to testshade or testrender, so maybe we should eliminate it, and not bother adding `--library`.

FYI, testshade and testrender do not really need separate command line options added for every possible new SS attribute... the `--options` flag can set them:

    testshade --options searchpath:library "blah/blah" ...other args...

So let's add new command line options for true options for those programs, or cases where many tests need to set them easily/commonly. But for settings that merely pass values directly to ShadingSystem attributes, especially if they are not commonly needed, I think we can rely on `--options` for simplicity.

Reply:
I understand for "--library %s" that you want to not list it.  OK.  Are you suggesting that we remove "--path %s" as well?

Reply:
Yes, I think we should get rid of both. The "path" one is particularly confusing now that there are two different kinds of searchpaths.

A lot of those command line options that merely set a value of a ShadingSystem option, but don't otherwise have any testshade-specific logic, were added to testshade before the --options catch-all flag existed. If I had thought to add --options first, perhaps most of them would never have been added, especially those that are rarely or never used in our testsuite.

Reply:
I've never used "library %s" option via command line, as the defaults do what I needed.  So probably a good reason.  Ok, I'll just remove both those command line options.